src/LibHdb++Cassandra.h

```diff
@@ -116,8 +81,14 @@ class HdbPPCassandra : public AbstractDB
      *      - local_dc: Datacenter name used for queries with LOCAL consistency
      *        level (e.g. LOCAL_QUORUM). In the current version of this library, all the
      *        statements are executed with LOCAL_QUORUM consistency level.
+     *      - store_diag_time: Either true to store the times or false to omit them.
      * - Debug:
-     *     - logging_enabled: Either true to enable command line debug, or false to disable
+     *     - logging_enabled: One of the following:
```

Comment:
I think it would make sense to rename logging_enabled to logging_level parameter because the name is suggesting this is a boolean as it used to be in previous versions.